lib/Doctrine/Migrations/Metadata/ExecutedMigration.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations\Metadata;
+
+use DateTime;
+use Doctrine\Migrations\Version\Version;
+
+class ExecutedMigration
+{
+    /** @var Version */
+    private $version;
+
+    /** @var DateTime */
+    private $executedOn;
+
+    /**
+     * Milliseconds
+     *
+     * @var int
+     */
+    public $executionTime;
+
+    public function __construct(Version $version, ?DateTime $executedOn, ?int $executionTime)
```

Comment:
ExecutionTime::fromMilliseconds(...) ?

Reply:
Hmm.. . And that method what should do? Just return back the same value?

Reply:
Yes but its self explanatory. Looking at `$executionTime` variable name is not helpful, unless it would be named `$executionTimeInMs` oder `$executionTimeInMilliseconds`.

it could also provide a `ExecutionTime->inSeconds()` which could be helpful when outputting it to the user? 🤔

Reply:
What you are suggesting is a value object instead of the integer, right?

Reply:
```php
final class ExecutionTime
{
    /** var int */
    private $time;

    private function __construct(int $time)
    {
        $this->time = $time;
    {

    public function fromMilliseconds(int $time): self
    {
        return new self($time);
    }

    public function toMilliseconds(): int
    {
        return $time;
    }

    public function toSeconds(): // int or float here
    {
        return $time / 1000;
    }
}
```

Something like this